trunk_merge.sh

```diff
@@ -23,6 +23,8 @@ echo "Detected merge queue commit, using HEAD^1 (${upstream}) as upstream and HE
 
 if [[ -n ${INPUT_CHECK_RUN_ID} ]]; then
   annotation_argument=--trunk-annotate=${INPUT_CHECK_RUN_ID}
+else
+  annotation_argument=--github-annotate
```

Comment:
```suggestion
  annotation_argument=""
```

not sure if github-annotate works on trunk_merge, we've never tested it afaik

Reply:
oh i totally didn't catch that my bad, idk why i did --github-annotate